hash/gba.xml

```diff
@@ -15794,6 +15829,18 @@ Hangs during cutscene after beating Maxim for best ending path (i.e. before real
 		</part>
 	</software>
 
+	<!-- Game crashes MAME immediately -->
+	<software name="ininjap" supported="no">
+		<description>I-Ninja (prototype)</description>
+		<year>2003</year>
+		<publisher>Namco</publisher>
+		<part name="cart" interface="gba_cart">
+			<dataarea name="rom" size="52428800">
+				<rom name="i-ninja (unlabeled prototype) (gameboy advance, unreleased).gba" size="52428800" crc="e0cf48d9" sha1="431f9cd0c2b5976bd3362844369e6377dfda77a8"/>
+			</dataarea>
```

Comment:
This is an odd size as well – what chips did the cartridge actually have?

Reply:
There were no photos of the board posted, just the dev cart, so I couldn't add that info to the XML, sorry.

Reply:
Overdump, actual size seems to be 33554432 (0x2000000)

Reply:
Can confirm that trimming this makes it work.